src/App.tsx

```diff
@@ -50,15 +40,15 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector />
               </div>
 
               <div className="block" data-cy="MainContent">
                 {!author && <p data-cy="NoSelectedUser">No user selected</p>}
 
-                {author && !loaded && <Loader />}
+                {author && isLoading && <Loader />}
```

Comment:
The conditional rendering of the 'Loader' component is correct. Ensure that the 'Loader' component is properly defined and imported.